src/singletons/Settings.hpp

```diff
@@ -194,6 +195,7 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting enableEmoteImages = {"/emotes/enableEmoteImages", true};
     BoolSetting animateEmotes = {"/emotes/enableGifAnimations", true};
     FloatSetting emoteScale = {"/emotes/scale", 1.f};
+    BoolSetting showUnlistedEmotes = {"/emotes/showUnlistedEmotes", false};
```

Comment:
This setting doesn't mention 7TV although 7TV is the only emote-provider that has unlisted emotes. I kept it to allow a smooth migration of settings.

Reply:
Let's at least rename the variable name - I'm fine with leaving the setting path the same.

`showUnlistedSevenTVEmotes`